blogs/2022/05/18/dev-container-cli.md

```diff
@@ -0,0 +1,131 @@
+---
+Order: 73
+TOCTitle: Dev container CLI
+PageTitle: Development Container CLI
+MetaDescription: Provide a consistent development environment anywhere using the development container CLI.
+Date: 2022-05-18
+Author: Brigit Murtaugh
+---
+
+# The dev container CLI
+**A consistent environment anywhere with the dev container CLI**
+
+May 18, 2022 by Brigit Murtaugh, [@BrigitMurtaugh](https://twitter.com/BrigitMurtaugh)
+
+## TLDR;
+
+We’ve released a dev container CLI as part of the new Development Containers Specification. We’d love your feedback in [this issue](https://github.com/devcontainers/cli/issues/7), or new issues and PRs in the repo.
+
+## Context
```

Comment:
Maybe a less generic header? How about "Consistency = joy"?
Or if that is too poppy, "A consistent, predictable environment"

Reply:
I like "Consistency = joy," it made me smile when I read that! I'll update.